Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/AtomicSequences/Validation/EconDamperPositionLimitsMultiZone_LoopDisable.mo

```diff
@@ -0,0 +1,183 @@
+within Buildings.Experimental.OpenBuildingControl.ASHRAE.G36.AtomicSequences.Validation;
+model EconDamperPositionLimitsMultiZone_LoopDisable
+  "Validation model for the multiple zone VAV AHU minimum outdoor air control - damper position limits"
+  extends Modelica.Icons.Example;
+
+  parameter Types.FreezeProtectionStage freProStage1 = Types.FreezeProtectionStage.stage1
+    "Freeze protection stage 1";
+  parameter Types.FreezeProtectionStage freProStage2 = Types.FreezeProtectionStage.stage2
+    "Freeze protection stage 2";
+  parameter Integer freProStage1Num = Integer(freProStage1)-1
```

Comment:
If I don't subtract it, we will have an integer value 1 for stage0, which, I thought might be confusing. Let me know if you'd still like me to use 1 for stage0, 2 for stage1, etc.